orcid-web/src/main/webapp/static/javascript/ng1Orcid/require.js

```diff
@@ -26,7 +26,6 @@ require("jquery");
 require("../jquery-migrate/1.3.0/jquery-migrate-1.3.0.min.js");
 require("../typeahead/0.9.3/typeahead.min.js");
 require("../script.js");
-require("../orcid.js");
```

Comment:
@lizkrznarich  Great we are removing this file. 

We can now remove it also from 

https://github.com/ORCID/ORCID-Source/blob/d7cd97b8be45081b59df704a4758bd669d532531/orcid-web/src/main/webapp/static/javascript/ng1Orcid/ci.webpack.config.js#L32
And all the other webpack files

I now that using a Regexp to map each script file by name on webpack the is not pretty but seems to be the only way.